spec/support/acceptance/adds_shoulda_matchers_to_project.rb

```diff
@@ -87,7 +87,8 @@ def library_config_for(libraries)
 
     def test_helper_file_for(test_framework, libraries)
       if integrates_with_rails?(test_framework, libraries) ||
-        integrates_with_nunit?(test_framework)
+        integrates_with_nunit?(test_framework) ||
+        integrates_with_rspec_expectations?(test_framework)
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.